### ISSUE TITLE
Nextcloud config.php file not found

### ISSUE THREAD

**Issue by newmediologo** [nextcloud/nextcloud-snap#676]
Hi.. I've reinstalled Nextcloud v. 13.0.5 and I want to change the datadir to the external drive as I have before as explained here: https://github.com/nextcloud/nextcloud-snap/issues/150

Now I can't find che config.php in: /var/snap/nextcloud/current/nextcloud/config/config.php, whe I go to edit it the file is empty..

There's a new location for the config.php file..? Where I can find it..?

Thank You very much

**Comment by kyrofa** [comment]
No, that file hasn't moved, but after initial install it takes a minute for it to show up there.

**Comment by newmediologo** [comment]
Thanks @kyrofa solved.. File were not visible when I've deactivate Nexcloud, after reactivate I've found and edit it.. Thanks..

**Comment by NoxNoscope** [comment]
what command did u use to get the file to appear?
